Models/DTOs/UpdateUsersAboutDTO.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace AonFreelancing.Models.DTOs
+{
+    public class UpdateUsersAboutDTO
```

Comment:
Avoid using verbs in class names, use nouns instead. For example, `UserAboutInputDTO`

Reply:
Done